tests/caldavclient/tst_caldavclient.cpp

```diff
@@ -181,15 +180,15 @@ void tst_CalDavClient::mergeAccountCalendars()
     client.mManager = mManager; // So we can share the same Account pointers.
     QVERIFY(client.init());
 
-    QList<PropFind::CalendarInfo> remoteCalendars;
-    remoteCalendars << PropFind::CalendarInfo{QLatin1String("/bar%40plop/"),
+    QList<Buteo::Dav::CalendarInfo> remoteCalendars;
+    remoteCalendars << Buteo::Dav::CalendarInfo{QLatin1String("/bar%40plop/"),
             QLatin1String("Bar"), QLatin1String("#0000FF"), QLatin1String("/principals/2")};
-    remoteCalendars << PropFind::CalendarInfo{QLatin1String("/foo/"),
+    remoteCalendars << Buteo::Dav::CalendarInfo{QLatin1String("/foo/"),
             QLatin1String("New foo"), QLatin1String("#FF0000"), QString()};
-    remoteCalendars << PropFind::CalendarInfo{QLatin1String("/toto%40tutu/"),
+    remoteCalendars << Buteo::Dav::CalendarInfo{QLatin1String("/toto%40tutu/"),
             QLatin1String("Toto"), QLatin1String("#FF00FF"), QString()};
 
-    const QList<PropFind::CalendarInfo> &calendars = client.mergeAccountCalendars(remoteCalendars);
+    const QList<Buteo::Dav::CalendarInfo> &calendars = client.mergeAccountCalendars(remoteCalendars);
```

Comment:
Executed the unit test and having a failure a few lines below on color comparison. Actual value looks like userPrincipal.